_includes/misc/advcode-shortcuts.md

```diff
@@ -0,0 +1,13 @@
+## Advanced Code Editor search and replace keyboard shortcuts
+
+Advanced Code Editor provides the following shortcuts for search and replace within the code dialog.
+
+| Action | PC | Mac |
+| ---- | ----- | ----- |
+| Find | Ctrl+F | Cmd+F |
+| Find next instance | Ctrl+G | Cmd+G |
+| Find previous instance | Shift+Ctrl+G | Shift+Cmd+G |
+| Replace | Ctrl+H | Cmd+H |
```

Comment:
I thought we said we couldn't document `Cmd+H` on Mac as it "hides" the window in Safari?

Reply:
@lnewson @trevorjay did we remap it to something else?

Reply:
It still has the original shortcut mentioned on the demo page, ie: `Shift+Cmd+Option+F`. Not the nicest though, but that's all I believe works on Safari (Chrome/Firefox aren't impacted). @trevorjay can confirm.

Reply:
I can confirm that the hideous `Shift+Cmd+Opt+F` does work on Safari.

Reply:
Wait, isn't that replaceAll? @lnewson @trevorjay

Reply:
Right, I copied the wrong shortcut sorry... should have been `Cmd+Option+F`

Reply:
`Cmd-Opt-F` is Replace and `Shift-Cmd-Opt-F` is replace all.

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1594/commits/42b7fcf5c90819e237ab7dccd63ff44883971ee0